### PR TITLE
MICS-14354 add batch update route

### PR DESCRIPTION
**Pull request by F-Jules**

### CODE REVIEW

File: src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts [hunk 1]
```diff
@@ -109,6 +112,10 @@ export abstract class AudienceFeedConnectorBasePlugin extends BasePlugin<Audienc
     instanceContext: AudienceFeedConnectorBaseInstanceContext
   ): Promise<UserSegmentUpdatePluginResponse>;
 
+  protected abstract onBatchUpdate(
+    request: BatchUpdateRequest<unknown>
```
Comment (nguillomon) [inline src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts:116]: May be you should pass instanceContext like others signature above ?

File: src/mediarithmics/api/plugin/audiencefeedconnector/AudienceFeedConnectorPluginResponseInterface.ts [hunk 2]
```diff
@@ -45,9 +45,29 @@ export interface ExternalSegmentConnectionPluginResponse {
...
+  feed_id: string;
+  feed_session_id: string;
+  segment_id: string;
+  datamart_id: string;
+}
+
+export interface BatchUpdatePluginResponse {
+  status: DeliveredDataPluginResponseStatus;
+  message?: string;
+  nextMsgDelayInMs?: number;
```
Comment (nguillomon) [inline src/mediarithmics/api/plugin/audiencefeedconnector/AudienceFeedConnectorPluginResponseInterface.ts:70]: should be in snake case  next_msg_delay_in_ms

File: src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts [hunk 3]
```diff
@@ -362,4 +369,71 @@ export abstract class AudienceFeedConnectorBasePlugin extends BasePlugin<Audienc
...
+          const response: BatchUpdatePluginResponse =
+            await this.onBatchUpdate(request);
+
+          this.logger.debug(`Returning: ${JSON.stringify(response)}`);
+
+          const pluginResponse: BatchUpdatePluginResponse = {
+            status: response.status,
+          };
+
+            if (response.nextMsgDelayInMs) {
```
Comment (nguillomon) [inline src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts:398]: identation

File: README.md [hunk 4]
```diff
@@ -163,11 +163,36 @@ Testing Plugins is highly recommended.
...
+  feed_id: string;
+  feed_session_id: string;
+  segment_id: string;
+  datamart_id: string;
+}
+
+export interface BatchUpdatePluginResponse {
+  status: DeliveredDataPluginResponseStatus;
+  message?: string;
+  nextMsgDelayInMs?: number;
```
Comment (nguillomon) [inline README.md:189]: Update snake case

File: src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts (hunk 1 above)
Comment (F-Jules) [reply src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts:116]: Make complete sense

**Pull request merged**